### PR TITLE
refactor: Remove ontology from KnoraProject entity

### PR DESCRIPTION
**Pull request by seakayone**
### Task Description/Number

Issue Number: DEV-

### PR Type

- [ ] build/chore: maintenance tasks (no production code change)
- [ ] docs: documentation changes (no production code change)
- [ ] feat: represents new features
- [ ] fix: represents bug fixes
- [ ] perf: performance improvements
- [x] refactor: represents production code refactoring
- [ ] test: adding or refactoring tests (no production code change)

### Basic requirements for bug fixes and features

- [ ] Tests for the changes have been added
- [ ] Docs have been added / updated

### Does this PR introduce a breaking change?

- [ ] Yes

### Does this PR change client-test-data?

- [ ] Yes

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/slice/admin/repo/service/KnoraProjectRepoLive.scala [hunk 1]
```diff
@@ -183,18 +93,96 @@ final case class KnoraProjectRepoLive(
...
+            |PREFIX owl: <http://www.w3.org/2002/07/owl#>
+            |CONSTRUCT {
+            |  ?project ?p ?o .
+            |} WHERE {
+            |  BIND(IRI("${iri.value}") as ?project)
+            |  ?project a knora-admin:knoraProject .
+            |  OPTIONAL {
+            |      ?ontology a owl:Ontology .
+            |      ?ontology knora-base:attachedToProject ?project .
+            |  }
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/slice/admin/repo/service/KnoraProjectRepoLive.scala:124]: maybe I'm fundamentally misunderstanding something (or am just tired so late in the evening), but why is it necessary to query for the ontology, if it's no longer added to the `CONSTRUCT` and then used in the `KnoraProject` object?

**Review by BalduinLandolt — CHANGES_REQUESTED** [review]
Correct me, if I'm wrong, but I think the ontology can be removed entirely from the queries, which would presumably make them faster, and definitely makes them shorter.

File: webapi/src/main/scala/org/knora/webapi/slice/admin/repo/service/KnoraProjectRepoLive.scala (hunk 1 above)
Comment (seakayone) [reply webapi/src/main/scala/org/knora/webapi/slice/admin/repo/service/KnoraProjectRepoLive.scala]: You have it right, removed the needless query parts in f26f56f35447b932d5beb4a56ed263eda4320702

**Review by BalduinLandolt — APPROVED** [review]
Nice, thanks!

**Review by mpro7 — APPROVED** [review]

**Pull request merged**